app/models/resource.py

```diff
@@ -0,0 +1,30 @@
+from .. import db
+
+
+class Resource(db.Model):
+    __tablename__ = 'resources'
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(64))
+    address = db.relationship('Address', backref='resource',
+                              lazy='dynamic', uselist=False)
+    description = db.Column(db.Text)
+    website = db.Column(db.Text)
+    reviews = db.relationship('Review', backref='resource',
+                              lazy='dynamic')
+
+    def __repr__(self):
+        return '<Resource \'%s\'>' % self.name
+
+
+class Address(db.Model):
+    __tablename__ = 'addresses'
+    id = db.Column(db.Integer, primary_key=True)
+    receipt_line = db.Column(db.Text)
```

Comment:
These technical names might be foreign to some people (including me). I like that we use them, but let's throw a comment on each line that's like `# 3900 Pine St.` or `# Mark's Hardware Store` to give an example.